ios/RCTWebRTC/WebRTCModule+DevicesManager.m

```diff
@@ -0,0 +1,301 @@
+//
+//  WebRTCModule+DevicesManager.m
+//  react-native-webrtc
+//
+//  Created by Filipi Fuchter on 08/03/22.
+//
+
+#import "WebRTCModule.h"
+
+@interface WebRTCModule (DevicesManager)
+
+@end
+
+@implementation WebRTCModule (DevicesManager)
+
+static NSString const *DEVICE_KIND_VIDEO_INPUT = @"videoinput";
+static NSString const *DEVICE_KIND_AUDIO_INPUT = @"audioinput";
+static NSString const *DEVICE_KIND_AUDIO_OUTPUT = @"audiooutput";
+
+enum DeviceType {
+    BLUETOOTH=1,
+    SPEAKER=2,
+    EARPIECE_HEADSET=3,
+    BUILT_IN_MICROPHONE=4,
```

Comment:
i guess if we only return input devices for audio from `enumerateDevices()` (as suggested in an earlier comment of mine) then we wouldn't need both `EARPIECE_HEADSET` and `BUILT_IN_MICROPHONE`, and both `SPEAKER` and `BUILT_IN_MICROPHONE_SPEAKER`, right?

Reply:
Right, but, I think we should follow the webrtc specification and return both `audioinput` and `audiooutput`, don't you think ?

Reply:
Fixed with the refactor that we have agreed to only return `audio`.